test/test_torch.py

```diff
@@ -10023,6 +10023,19 @@ def run_subtest(guess_rank, actual_rank, matrix_size, batches, device, pca, **op
         guess_rank, actual_rank, size, batches = 2, 2, (17, 4), ()
         run_subtest(guess_rank, actual_rank, size, batches, device, jitted)
 
+    @onlyCPU
```

Comment:
All the svd tests should go in test_linalg.py, even the ones for torch.svd

Reply:
done in 5fa4efe329.
However it is a bit weird now: there is `test_linalg.py:test_svd` which does NOT test `torch.linalg.svd` but it tests `torch.svd`.
I tried to improve the situation by renaming the new tests into `test_linalg_svd_*` and to put comments to divide the two sections.